integration_test/commons/test_util.dart

```diff
@@ -7,11 +7,30 @@
 
 import 'dart:convert';
 import 'dart:math';
+import 'package:autonomy_flutter/common/injector.dart';
+import 'package:autonomy_flutter/service/configuration_service.dart';
+import 'package:autonomy_flutter/util/constants.dart';
 
+import '../../lib/main.dart' as app;
+
+import 'package:autonomy_flutter/main.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:http/http.dart' as http;
 
+final Finder autonomyheader = find.text('AUTONOMY');
```

Comment:
can we change the name convention into `start with a lowercase letter and then capitalize the first letter of every subsequent word`

Reply:
I changed the name of variables to our convention.